src/lib/Form/Factory/FormFactory.php

```diff
@@ -1018,18 +1023,22 @@ public function forgotUserPasswordWithLogin(
     /**
      * @param \EzSystems\EzPlatformAdminUi\Form\Data\User\UserPasswordResetData $data
      * @param null|string $name
+     * @param null|User $user
      *
      * @return \Symfony\Component\Form\FormInterface
      *
      * @throws \Symfony\Component\OptionsResolver\Exception\InvalidOptionsException
      */
     public function resetUserPassword(
         UserPasswordResetData $data = null,
-        ?string $name = null
+        ?string $name = null,
+        ?User $user = null
```

Comment:
I think we are passing additional arguments as options, but I am not going to force this one.